pod/perlsyn.pod

```diff
@@ -549,6 +558,24 @@ followed by C<my>.  To use this form, you must enable the C<refaliasing>
 feature via C<use feature>.  (See L<feature>.  See also L<perlref/Assigning
 to References>.)
 
+As of Perl 5.36, you can iterate over a list of lexical scalars n-at-a-time.
+You can only iterate over scalars - unlike list assignment, it's not
+possible to use C<undef> to signify a value that isn't wanted.  This is a
+limitation of the current implementation, and might be changed in the
+future.
+
+If the size of the LIST is not an exact multiple of number of iterator
```

Comment:
`not an exact multiple of number of iterator` -> `not an exact multiple of the number of iterator`

Reply:
D'oh!

Well spotted. Fixed.